edge-modules/azure-monitor/docker/windows/amd64/Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG base_tag=3.1.12-nanoserver-1809
```

Comment:
Windows containers aren't supported in 1.2...you can remove this Dockerfile.

Reply:
Removed.